roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1056,6 +1056,7 @@ def get_defaults(self, roles):
                       public_hostname=hostname)
         common['client_binary'] = 'oc' if os.path.isfile('/usr/bin/oc') else 'osc'
         common['admin_binary'] = 'oadm' if os.path.isfile('/usr/bin/oadm') else 'osadm'
+        common['dns_domain'] = 'cluster.local'
```

Comment:
With these added to common, we can kill the dns role in openshift_facts with fire and update roles/openshift_master/tasks/main.yml accordingly.